third_party/closure-responding-channel/closure-wrapper.js

```diff
@@ -41,4 +45,15 @@ function createRC(frameWindow, origin, serviceHandlersMap) {
   return respondingChannel;
 }
 
+function createPO() {
```

Comment:
nit: can you give it the full name? `createPortOperator`

Reply:
Agree. Done.